### PR TITLE
Displaying ... when comp name more than 13 chars

### LINKED ISSUE #64: [Feature] Fixing component name design
**Is your feature request related to a problem? Please describe.**
If a component has a big name, the name exceeds the component size and goes for a walk on the HTML canvas.

**Describe the solution you'd like**
If a component has a name bigger than a number of letters, display ... after the number.
F.ex.,
Bananabananabananabanana
=> Bananabanan...

### PR DESCRIPTION
**Pull request by Evgenios95**
Implemented simple function that returns the component name and ... in case the name is too big.

Screenshot 2022-04-16 at 13 25 54

closes #64

### CODE REVIEW

File: packages/bratus-app/src/components/ComponentNode/ComponentNode.jsx [hunk 1]
```diff
@@ -98,6 +98,14 @@ const ComponentNode = (node) => {
   const layoutSourceHandlePosition =
     treeLayoutDirection === 'LR' ? 'right' : 'bottom';
 
+  const truncateNodeName = (nodeName, nameLength) => {
+    return nodeName.length > nameLength
+      ? nodeName.slice(0, nameLength - 1).concat('...')
+      : nodeName;
+  };
+
+  const truncatedNodeName = truncateNodeName(node.data.label, 13);
```
Comment (thepiotrek) [inline packages/bratus-app/src/components/ComponentNode/ComponentNode.jsx:107]: Nice and easy.
Should the value 13 maybe be placed in the constants file? 
truncatedNodeLength , or similar? :)

**Review by thepiotrek — APPROVED** [review]
I just had a super minor comment.

File: packages/bratus-app/src/components/ComponentNode/ComponentNode.jsx (hunk 1 above)
Comment (Evgenios95) [reply packages/bratus-app/src/components/ComponentNode/ComponentNode.jsx:107]: For sure! I'll do that ☺️

Comment (Evgenios95) [reply packages/bratus-app/src/components/ComponentNode/ComponentNode.jsx]: Resolved

**Pull request merged**